provisioning/src/handlers/portfolioDrafts/portfolioDrafts.post.ts

```diff
@@ -12,11 +12,11 @@ export async function postPortfolioDrafts (context: Context, req: HttpRequest):
     const { database } = await client.databases.createIfNotExists({ id: "atat" });
     const { container } = await database.containers.createIfNotExists({ id: "portfolios" });
 
-    // TODO: generate timestamps
+    let now = JSON.stringify(new Date());
```

Comment:
Better as `let now = new Date().toISOString()`?  That's what underlying stringify.